web/src/app/users/UserContactMethodEditDialog.tsx

```diff
@@ -96,3 +102,15 @@ export default function UserContactMethodEditDialog({
     />
   )
 }
+
+export default function UserContactMethodEditDialogSwitch(
+  props: UserContactMethodEditDialogProps,
+): React.ReactNode {
+  const isDestTypesSet = useExpFlag('dest-types')
+
+  if (isDestTypesSet) {
+    return <UserContactMethodEditDialogDest {...props} />
```

Comment:
something in here is giving the following console error:
`Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.`

Reply:
issue was due to nesting another component in Typography component in UserContactMethodListDest.tsx. Pushed Fix